docs/DeveloperGuide.md

```diff
@@ -1228,27 +1200,15 @@ testers are expected to do more *exploratory* testing.
 
    1. Prerequisites: Be on the home screen using the `home` command. Contact list should be displayed in the middle.
 
-   1. Test case: `delete_contact 2`<br>
-      Expected: Second contact is deleted from the list. Details of the deleted contact shown in the status message.
-
-   1. Test case: `delete_contact 0`<br>
-      Expected: No contact is deleted. Error details shown in the status message.
-
-   1. Other incorrect delete commands to try: `delete_contact`, `delete_contact x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+   1. Test cases and expected outcomes are similar to the previous case.
 
 1. Deleting a contact after selecting an event
 
    1. Prerequisites: Be on the selected event page using the `select_event x` command in which x represents an integer that is smaller than or equal to the event list size. Multiple contacts in the list of contacts linked to the event.
 
-   1. Test case: `delete_contact 1`<br>
-      Expected: First contact in the event is deleted from the entire list. Returns back to home page, with the full list showing that the contact has been deleted. Details of the deleted contact shown in the status message.
-
-   1. Test case: `delete_contact 0`<br>
-      Expected: No contact is deleted. Error details shown in the status message.
+   1. Test cases and expected outcomes are similar to the previous case except for that the index is based on the currently displayed contact list associated to the selected event.
 
-   1. Other incorrect delete commands to try: `delete_contact`, `delete_contact x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+4. Deleting a contact on other pages causes an error since the contact list is not being displayed.
```

Comment:
Perhaps change this to a note box instead?

Reply:
good suggestion!